client/my-sites/checkout/composite-checkout/test/composite-checkout-thank-you.js

```diff
@@ -933,7 +933,29 @@ describe( 'getThankYouPageUrl', () => {
 		expect( url ).toBe( redirectTo );
 	} );
 
-	it( 'redirects to the jetpack checkout thank you when jetpack checkout arg is set', () => {
+	it( 'redirects to the redirects to the user connection/authorization page when jetpack checkout arg is set and the adminUrl is set', () => {
```

Comment:
I think you added an extra `redirects to the` to the test description. 🙂

Reply:
Oh yep! 🤦‍♂️  Thanks!